src/applications/gi/components/AccordionItem.jsx

```diff
@@ -20,14 +20,16 @@ class AccordionItem extends React.Component {
     const expanded = this.state.expanded;
     return (
       <li>
-        <button
-          onClick={this.toggle}
-          className="usa-accordion-button"
-          aria-expanded={expanded}
-          aria-controls={this.id}
-        >
-          <h2>{this.props.button}</h2>
-        </button>
+        <h2 className="accordion-button-wrapper">
```

Comment:
can probably use the `.vads-u-margin--0` [margin utility](https://design.va.gov/utilities/margins) from the design system instead of adding new css.